ncard_app/admin.py

```diff
@@ -7,8 +7,8 @@
 admin.site.register(models.Award)
 admin.site.register(models.Event)
 admin.site.register(models.Publication)
-admin.site.register(models.Country)
-admin.site.register(models.PersonAddress)
+
```

Comment:
models.Country and models.PersonAddress should still be accessible in the admin site, even if they're not accessible in the front end.

Reply:
I already add that in my branch